release notes/v0.35.0.md

```diff
@@ -0,0 +1,94 @@
+k6 v0.35.0 is here! :tada: It introduces several new features that nicely enhance usage of k6 as well as contains a whole number of fixes and ongoing efforts with refactoring.
+
+In total, we have closed 14 issues and merged 43 PRs. We have also branched out three new xk6 extensions during this release :star:
+
+## New features
+
+### VU tags ([#2172](https://github.com/grafana/k6/pull/2172))
+
+k6 now supports setting tags for VUs as part of the Execution API with an easy key-value interface. Example usage:
+
+```js
+import exec from 'k6/execution';
+import { sleep } from 'k6';
+
+export const options = {
+    duration: '10s',
+    vus: 3,
+};
+
+export default function () {
+    exec.vu.tags['mytag'] = 'value';
+    exec.vu.tags['vuId'] = exec.vu.idInTest;
+    console.log("mytag is", exec.vu.tags['mytag'], 'and my VU\'s ID in tags', exec.vu.tags['vuId']);
```

Comment:
```suggestion
    console.log(`mytag is ${exec.vu.tags['mytag']}, and my VU's ID in tags ${exec.vu.tags['vuId']}`;
```
Sorry for spotting it only now, but I think a template literal could improve it.

Reply:
Yes, I agree; ~ "get used to JS fancy stuff" :smile: